jax/_src/numpy/lax_numpy.py

```diff
@@ -6690,22 +6690,43 @@ class _IndexUpdateHelper:
 
   The ``at`` property is syntactic sugar for calling the indexed update functions
   defined in :mod:`jax.ops`, and acts as a pure equivalent of in-place
-  modificatons. For further information, see `Indexed Update Operators
-  <https://jax.readthedocs.io/en/latest/jax.ops.html#indexed-update-operators>`_.
+  modificatons.
 
   In particular:
 
-  - ``x = x.at[idx].set(y)`` is a pure equivalent of ``x[idx] = y``.
-  - ``x = x.at[idx].add(y)`` is a pure equivalent of ``x[idx] += y``.
-  - ``x = x.at[idx].multiply(y)`` (aka ``mul``) is a pure equivalent of
-    ``x[idx] *= y``.
-  - ``x = x.at[idx].divide(y)`` is a pure equivalent of ``x[idx] /= y``.
-  - ``x = x.at[idx].power(y)`` is a pure equivalent of ``x[idx] **= y``.
-  - ``x = x.at[idx].min(y)`` is a pure equivalent of
-    ``x[idx] = minimum(x[idx], y)``.
-  - ``x = x.at[idx].max(y)`` is a pure equivalent of
-    ``x[idx] = maximum(x[idx], y)``.
+  ==============================  ================================
+  Alternate syntax                Equivalent In-place expression
+  ==============================  ================================
+  ``x = x.at[idx].set(y)``        ``x[idx] = y``
+  ``x = x.at[idx].add(y)``        ``x[idx] += y``
+  ``x = x.at[idx].multiply(y)``   ``x[idx] *= y``
+  ``x = x.at[idx].divide(y)``     ``x[idx] /= y``
+  ``x = x.at[idx].power(y)``      ``x[idx] **= y``
+  ``x = x.at[idx].min(y)``        ``x[idx] = minimum(x[idx], y)``
+  ``x = x.at[idx].max(y)``        ``x[idx] = maximum(x[idx], y)``
+  ``x = x.at[idx].get()``         ``x = x[idx]``
+  ==============================  ================================
+
+  None of these expressions modify the original ``x``; instead they return
+  a modified copy of ``x``. However, inside a :py:func:`jax.jit` compiled function,
+  expressions like ``x = x.at[idx].set(y)`` are guaranteed to be applied in-place.
+
+  Unlike NumPy in-place operations such as :code:`x[idx] += y`, if multiple
+  indices refer to the same location, all updates will be applied (NumPy would
+  only apply the last update, rather than multiplying the updates.) The order
```

Comment:
"multiplying" only makes sense for `index_mul` perhaps. But instead "applying" might work.

Reply:
Done.